spec/menu_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'menu'
+
+include Menu
+
+describe Menu do 
+
+	it 'has a hash with all the dishes' do 
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Tab detected.<br>Trailing whitespace detected.